app/src/main/java/com/gerardojim/nuggetscalculator/ui/main/viewmodel/CalculateViewModel.kt

```diff
@@ -13,7 +13,7 @@ import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.flow.consumeAsFlow
 import kotlinx.coroutines.launch
 
-class MainViewModel(private val calculator: DoggyMealCalculator) : ViewModel() {
+class CalculateViewModel(private val calculator: DoggyMealCalculator) : ViewModel() {
     val userIntent = Channel<MainIntent>(Channel.UNLIMITED)
     private val viewState = MutableStateFlow<MainState>(MainState.Init())
     val state: StateFlow<MainState> get() = viewState
```

Comment:
State is manipulated through the view models?!

Reply:
Techincally. The ViewModel spits out a ViewState